lib/utils/standalone.js

```diff
@@ -13,6 +13,9 @@ const platform = (() => {
   }
 })();
 const arch = (() => {
+  if (process.arch === 'arm64' && process.platform === 'darwin') {
```

Comment:
It'll be cleaner to include that logic in below switch statement

Reply:
Changed, I wanted to highlight that edge case and handle it first, but you're right that it could've been done in a cleaner way